Zilon.Core/Zilon.Core.Tests/Persons/MonsterPersonTests.cs

```diff
@@ -59,8 +60,8 @@ public void Constructor_HpInScheme_ActorHpEqualsSchemeHp()
             // ACT
             var monster = new MonsterPerson(monsterScheme);
 
-            // ARRANGE
-            monster.Hp.Should().Be(expectedHp);
+            //// ARRANGE
+            //monster.Hp.Should().Be(expectedHp);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5480188)